src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -157,6 +163,7 @@ def load_arguments(self, _):
         c.argument('sku', arg_type=snapshot_sku)
         c.argument('incremental', arg_type=get_three_state_flag(), min_api='2019-03-01',
                    help='Whether a snapshot is incremental. Incremental snapshots on the same disk occupy less space than full snapshots and can be diffed')
+        c.ignore('edge_zone')
```

Comment:
why do you need to ignore the variable here?

Reply:
Python SDK is not released.